tests/ui/suggestions/trait-impl-param-mismatched-cross-crate.rs

```diff
@@ -0,0 +1,40 @@
+//@ aux-build:trait-impl-params.rs
+//! Tests cross-crate trait impl param mismatch suggestions (keep aliases when matching,
+//! append missing; replace otherwise).
+//! regression test for https://github.com/rust-lang/rust/issues/106999
+extern crate trait_impl_params;
```

Comment:
Could you add `//@ run-rustfix`? That would help ensure that the suggestions are actually helpful.